### PR TITLE
Feature/table radio button 2

### PR DESCRIPTION
**Pull request by adamhaeger**
## Description

- Adds RadioButton and Datepicker functionality to SimpleTable

## Related Issue(s)

- closes #{issue number}

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [x] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [x] Unit test(s) have been added/updated
  - [ ] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [x] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [ ] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [ ] No functionality has been changed/added, so no documentation is needed
  - [x] I will do that later/have created an issue
  
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [ ] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [ ] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [ ] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

File: src/app-components/Datepicker/Datepicker.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import { DatePickerInput } from 'src/layout/Datepicker/DatePickerInput';
+import { getSaveFormattedDateString } from 'src/utils/dateHelpers';
+
+export type DatePickerControlProps = {
+  id: string;
+  value: string;
+  dateFormat: string;
+  timeStamp?: boolean;
+  onValueChange: (isoDateString: string) => void;
+  onSelectDate?: (date: Date) => void;
```
Comment (paal2707) [inline src/app-components/Datepicker/Datepicker.tsx:20]: Snakket vi om å fjerne denne?

File: src/app-components/Datepicker/Datepicker.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  isMobile?: boolean;
+};
+
+export const DatePickerControl: React.FC<DatePickerControlProps> = ({
+  id,
+  value,
+  dateFormat,
+  timeStamp = true,
+  onValueChange,
+  onSelectDate,
```
Comment (paal2707) [inline src/app-components/Datepicker/Datepicker.tsx:35]: fjerne denne?

File: src/layout/AddToList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  getDisplayData(_: BaseLayoutNode<'AddToList'>, __: DisplayDataProps): string {
+    return '';
+  }
+  renderSummary(_: SummaryRendererProps<'AddToList'>): JSX.Element | null {
+    return <div>summary</div>;
+  }
+  render = forwardRef<HTMLElement, PropsFromGenericComponent<'AddToList'>>(
+    function LayoutComponentAddToListRender(props, _): JSX.Element | null {
+      return <AddToListComponent {...props}></AddToListComponent>;
```
Comment (paal2707) [inline src/layout/AddToList/index.tsx:31]: <AddToListComponent {...props} /> ??

File: src/layout/SimpleTable/SimpleTableComponent.tsx [hunk 4]
```diff
@@ -87,33 +53,122 @@ export function SimpleTableComponent({ node }: TableComponentProps) {
           callback: (_) => true,
         });
       },
-      buttonText: <Lang id='general.delete' />,
+      buttonText: <Lang id={'general.delete'} />,
```
Comment (paal2707) [inline src/layout/SimpleTable/SimpleTableComponent.tsx:56]: trenger ikke curlys her?

File: src/layout/SimpleTable/SimpleTableComponent.tsx [hunk 5]
```diff
@@ -87,33 +53,122 @@ export function SimpleTableComponent({ node }: TableComponentProps) {
...
     });
   }
 
+  if (item.enableEdit) {
+    actionButtons.push({
+      onClick: (idx, _) => {
+        setEditItemIndex(idx);
+        setShowEdit(true);
+      },
+      buttonText: <Lang id={'general.edit'} />,
```
Comment (paal2707) [inline src/layout/SimpleTable/SimpleTableComponent.tsx:68]: samme her?

File: src/layout/SimpleTable/SimpleTableComponent.tsx [hunk 6]
```diff
@@ -87,33 +53,122 @@ export function SimpleTableComponent({ node }: TableComponentProps) {
...
+                    handleChange(nextValue, rowIndex);
+                  }}
+                  schema={schema}
+                />
+              )
+            : undefined,
+        }))}
+        mobile={isMobile}
+        actionButtons={actionButtons}
+        actionButtonHeader={<Lang id={'general.action'} />}
```
Comment (paal2707) [inline src/layout/SimpleTable/SimpleTableComponent.tsx:170]: trenger ikke curlys her

**Review by paal2707 — APPROVED** [review]
Ser veldig bra ut, par kommentarer og mest nitpicking bare. Bra jobba!

File: src/layout/AddToList/index.tsx (hunk 3 above)
Comment (adamhaeger) [reply src/layout/AddToList/index.tsx]: Dette er en ekspremientell komponent, ligger bak feature flag :)

**Pull request merged**